rllib/algorithms/callbacks.py

```diff
@@ -446,24 +446,21 @@ def on_episode_end(
         episode.custom_metrics["tracemalloc/worker/vms"] = worker_vms
 
 
-def make_multi_callbacks(callback_class_list: List[Type[DefaultCallbacks]]):
+def make_multi_callbacks(
+    callback_class_list: List[Type[DefaultCallbacks]],
+) -> DefaultCallbacks:
     """Allows combining multiple sub-callbacks into one new callbacks class.
 
-    Example:
-
-        .. code-block:: python
-
-            config.callbacks(make_multi_callbacks([
-                MyCustomStatsCallbacks,
-                MyCustomVideoCallbacks,
-                MyCustomTraceCallbacks,
-                ....
-            ]))
+    The resulting DefaultCallbacks will call all the sub-callbacks' callbacks
```

Comment:
Let's just keep AlgorithConfig out of the picture.
The example we would have to construct to make this a copy/paste thing would be quite large.